src/page-modules/assistant/server/journey-planner/index.ts

```diff
@@ -156,6 +166,26 @@ export function createJourneyApi(
       return nonTransits;
     },
 
+    async lines(input) {
+      const potential = getLinesIfCached(input as LineInput);
+      if (potential) return potential;
+
+      const result = await client.query<LinesQuery, LinesQueryVariables>({
+        query: LinesDocument,
+        variables: { authorities: input.authorities },
```

Comment:
This can be cached with using ApolloClient directly without having to do your own caching
```suggestion
        variables: { authorities: input.authorities },
        fetchPolicy: 'cache-first'
```